### PR TITLE
add router

### PR DESCRIPTION
**Pull request by Gregfanyan**
Added basic react router

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
 import './App.css';
+import Routes from './Routes'
 
 function App() {
   return (
     <div className="App">
```
Comment (Crushford) [inline src/App.js]: are you still using this css?

File: src/Routes/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    return(
+    <Switch>
+        <Route exact path ="/" component={Home}/>
+        <Route exact path ='/header' component={Header}/>
+
+    </Switch>
+    )
+}
+
+export default Routes
```
Comment (Crushford) [inline src/Routes/index.js]: needs an empty line at end of file

File: src/Routes/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import {Switch, Route} from 'react-router-dom';
+
+import Home from '../components/Home';
+import Header from '../components/Header';
+
+const Routes =()  =>{
+    return(
+    <Switch>
+        <Route exact path ="/" component={Home}/>
+        <Route exact path ='/header' component={Header}/>
```
Comment (Crushford) [inline src/Routes/index.js]: why is this single quote and the above double quote?
are you using a code formatter?

File: src/Routes/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {Switch, Route} from 'react-router-dom';
+
+import Home from '../components/Home';
+import Header from '../components/Header';
+
+const Routes =()  =>{
+    return(
+    <Switch>
+        <Route exact path ="/" component={Home}/>
```
Comment (Crushford) [inline src/Routes/index.js]: Can you please also add the path for a restaurant? this should include an id in the url

**Review by Crushford — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (Gregfanyan) [reply src/App.js]: thanks for review. removed

File: src/Routes/index.js (hunk 2 above)
Comment (Gregfanyan) [reply src/Routes/index.js]: fixed

File: src/Routes/index.js (hunk 3 above)
Comment (Gregfanyan) [reply src/Routes/index.js]: done

File: src/Routes/index.js (hunk 4 above)
Comment (Gregfanyan) [reply src/Routes/index.js]: done

File: src/Routes/index.js (hunk 3 above)
Comment (Gregfanyan) [reply src/Routes/index.js]: fixed

**Review by Crushford — APPROVED** [review]

**Comment by Crushford** [comment]
looks good to me! well done!

**Comment by Gregfanyan** [comment]
> looks good to me! well done!

great. thanks:)

File: src/components/Home/index.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+
+import Restaurants from "../Restaurants";
+
+function Home() {
+  const [restaurants, setRestaurants] = useState([]);
+
+  useEffect(() => {
```
Comment (typeoh) [inline src/components/Home/index.js:9]: Nice use of useEffect here

**Review by typeoh — APPROVED** [review]
LGTM

**Pull request merged**